### PR TITLE
[infra] Update package.json

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit updates following packages
- Add `ts-node` : this will be used in Github Action to run infra/**/*.ts
- Upgrade `mocha` : from major version 8 to 9
- Remove `vscode-test` : never used

ONE-vscode-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,21 +101,20 @@
   },
   "devDependencies": {
     "@types/glob": "^7.1.3",
-    "@types/mocha": "^8.0.4",
     "@types/node": "^12.11.7",
     "@types/vscode": "^1.46.0",
     "@typescript-eslint/eslint-plugin": "^4.9.0",
     "@typescript-eslint/parser": "^4.9.0",
+    "@vscode/test-electron": "^2.1.3",
```
Comment (hyunsik-yoon) [inline package.json:108]: reference: https://www.npmjs.com/package/vscode-test

**Review by hyunsik-yoon — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (llFreetimell) [reply package.json:108]: Hmm... I think this package is never used @.@
Sorry for bothering you...

File: package.json [hunk 2]
```diff
@@ -113,9 +112,8 @@
     "glob": "^7.1.6",
     "htmlhint": "^0.15.2",
     "ini": "^2.0.0",
-    "mocha": "^8.1.3",
```
Comment (llFreetimell) [inline package.json:116]: @YongseopKim According to #328, should I keep `mocha` as it is?

Comment (YongseopKim) [reply package.json:116]: Yes. `mocha` will be used. Please do not remove this.

**Comment by YongseopKim** [comment]
I don't think package-lock.json is included.

**Comment by YongseopKim** [comment]
> I don't think package-lock.json is included.

After getting some info, now I think package-lock.json is necessary.

**Review by YongseopKim — APPROVED** [review]
LGTM

**Review by hyunsik-yoon — APPROVED** [review]

**Pull request merged**